util/transaction_test_util.cc

```diff
@@ -185,12 +185,14 @@ bool RandomTransactionInserter::DoInsert(DB* db, Transaction* txn,
       }
       bytes_inserted_ += key.size() + sum.size();
     }
-    ROCKS_LOG_DEBUG(db->GetDBOptions().info_log,
-                    "Insert (%s) %s snap: %" PRIu64 " key:%s value: %" PRIu64
-                    "+%" PRIu64 "=%" PRIu64,
-                    txn->GetName().c_str(), s.ToString().c_str(),
-                    txn->GetSnapshot()->GetSequenceNumber(), full_key.c_str(),
-                    int_value, incr, int_value + incr);
+    if (txn != nullptr) {
```

Comment:
I resorted to using `if (txn != nullptr)` instead of `assert(txn)` in these fixes as it seems like txn could be null here based on how checks are done in lines 171 and 199.